### PR TITLE
Setup Danger + Fix Bitrise

### PR DESCRIPTION
**Pull request by worker8**
### Overview
Setup [Danger](https://github.com/danger/danger) in Bitrise, so that it runs Android Lint using [this](https://github.com/loadsmart/danger-android_lint) and complains back to Github pull request.

### CODE REVIEW

**Comment by worker8** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: syntax error, unexpected tIVAR, expecting keyword_do or `{` or `(`
...t GIT_REPOSITORY_URL=git@github.com:worker8/Pixels.git
...                        ^~~~~~~. Updating the Danger gem might fix the issue. Your Danger version: 5.7.0, latest Danger version: 5.11.0

```
 #  from Dangerfile:1
 #  -------------------------------------------
 >  export GIT_REPOSITORY_URL=git@github.com:worker8/Pixels.git
 #  
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/src/main/res/layout/home_item.xml [hunk 1]
```diff
@@ -12,22 +12,47 @@
...
+        android:layout_marginStart="7dp"
+        android:alpha="0.7"
+        android:fontFamily="sans-serif-condensed"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toBottomOf="@id/homeItemUsername"
+        app:layout_constraintStart_toEndOf="@id/homeItemUsername"
+        app:layout_constraintTop_toTopOf="@id/homeItemUsername"
+        tools:text="16 hours ago" />
 
     <ImageView
```
Comment (worker8) [inline app/src/main/res/layout/home_item.xml]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/73c22a7f0c6c2d66ca9746b78b1ff215d5429583/app/src/main/res/layout/home_item.xml#L46"></span>Missing <code>contentDescription</code> attribute on image</td>
    </tr>
  </tbody>
</table>

Comment (worker8-1) [inline app/src/main/res/layout/home_item.xml]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43ff13c8bf415f17743a21d636736be69e6bb8df/app/src/main/res/layout/home_item.xml#L46"></span>Missing <code>contentDescription</code> attribute on image</td>
    </tr>
  </tbody>
</table>

File: app/build.gradle [hunk 2]
```diff
@@ -26,6 +26,12 @@ android {
         resolutionStrategy.force "com.android.support:animated-vector-drawable:$Versions.support"
```
Comment (worker8-1) [inline app/build.gradle]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43ff13c8bf415f17743a21d636736be69e6bb8df/app/build.gradle#L26"></span>All com.android.support libraries must use the exact same version specification (mixing versions can lead to runtime crashes). Found versions 28.0.0, 27.1.0. Examples include <code>com.android.support:animated-vector-drawable:28.0.0</code> and <code>com.android.support:exifinterface:27.1.0</code></td>
    </tr>
  </tbody>
</table>

**Pull request merged**